### ISSUE TITLE
Unable to see a message while playing on low resolution.

### ISSUE THREAD

**Issue by WSDguy2014** [cdqwertz/rpgtest#51]
I can't see a message while using ability. Tested in resolution: 854 480

**Comment by cdqwertz** [comment]
Which type of message? A chat message?

**Comment by WSDguy2014** [comment]
No, i a central_message, for example: using ability, texts will appear on central message, not on chat.
